stablehlo/transforms/Passes.td

```diff
@@ -144,3 +144,40 @@ def ShapeLegalizeToStablehloPass : Pass<"shape-legalize-to-stablehlo", "func::Fu
   }];
   let dependentDialects = ["mlir::stablehlo::StablehloDialect"];
 }
+
+def StablehloReplaceCompositesWithCallsPass :
+    Pass<"stablehlo-replace-composites-with-calls", "func::FuncOp"> {
```

Comment:
Brainstorming pass names:

```
--stablehlo-inline-composites
--stablehlo-decompose-composite
--stablehlo-use-composite-decompositions
--stablehlo-composite-use-decompositions
```

I'm OK with `inline-composites` even if its a bit of a misnomer. This current name is a bit verbose / implementation-detail-y. I.e. `replace-composites-with-call-to-decomposition` should be the true verbose name. I think `use-decomposition` may be more accurate?

Reply:
Thanks for taking the time to brainstorm names! I think `inline` and `decompose` are both misnomers, although maybe not majorly so. But as a user, if I called a pass called "inline" and it didn't actually inline the thing, I'd be confused.

I don't really understand what `use` means in this context, so e.g. `use-composite-decompositions` is a bit confusing to me.

I agree the current name might be too specific, but it's accurate. I like `replace-composites-with-call-to-decomposition` though.

So, I don't know. I would personally go with `replace-composites-with-call-to-decomposition`. WDYT?

Reply:
`--stablehlo-composite-call-decompositions` ?

Reply:
Discussed offline. We will go with `stablehlo-legalize-composite-to-call`, to be consistent with existing passes with names like `mhlo-legalize-X-to-Y`, e.g. `mhlo-legalize-dot-to-dot-general`.